.github/workflows/cd.yml

```diff
@@ -168,122 +121,119 @@ jobs:
-            if [ "${{ matrix.build.OS }}" = "windows-2022" ]; then
+            if [ "${{ contains(matrix.os, 'windows') }}" = "true" ]; then
               bin="${bin}.exe"
             fi
-            cp "target/${{ matrix.build.TARGET }}/release/${bin}" release/
+            cp "target/${{ matrix.target }}/release/${bin}" release/
           done
           mv release/ git-cliff-${{ env.RELEASE_VERSION }}/
       - name: Create release artifacts
         shell: bash
         run: |
-          if [ "${{ matrix.build.OS }}" = "windows-2022" ]; then
-            7z a -tzip "git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}.zip" \
+          if [ "${{ contains(matrix.os, 'windows') }}" = "true" ]; then
+            7z a -tzip "git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}.zip" \
               git-cliff-${{ env.RELEASE_VERSION }}/
           else
-            tar -czvf git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}.tar.gz \
+            tar -czvf git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}.tar.gz \
               git-cliff-${{ env.RELEASE_VERSION }}/
-            shasum -a 512 git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}.tar.gz \
-              > git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}.tar.gz.sha512
+            shasum -a 512 git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}.tar.gz \
+              > git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}.tar.gz.sha512
           fi
       - name: Sign the release
-        if: matrix.build.OS == 'ubuntu-22.04' || matrix.build.OS == 'macos-14'
+        if: contains(matrix.os, 'ubuntu') || contains(matrix.os, 'macos')
         run: |
           echo "${{ secrets.GPG_RELEASE_KEY }}" | base64 --decode > private.key
           echo "${{ secrets.GPG_PASSPHRASE }}" | gpg --pinentry-mode=loopback \
             --passphrase-fd 0 --import private.key
           echo "${{ secrets.GPG_PASSPHRASE }}" | gpg --pinentry-mode=loopback \
             --passphrase-fd 0 --detach-sign \
-            git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}.tar.gz
+            git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}.tar.gz
       - name: Publish to GitHub
         if: ${{ !contains(github.ref, '-') }}
         uses: svenstaro/upload-release-action@v2
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.build.TARGET }}*
+          file: git-cliff-${{ env.RELEASE_VERSION }}-${{ matrix.target }}*
           file_glob: true
           overwrite: true
           tag: ${{ github.ref }}
           release_name: "Release v${{ env.RELEASE_VERSION }}"
           body: "${{ needs.generate-changelog.outputs.release_body }}"
       - name: Publish to GitHub (pre-release)
-        if: ${{ contains(github.ref, '-') }}
+        if: contains(github.ref, '-')
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif
>When you use expressions in an if conditional, you can, optionally, omit the ${{ }} expression syntax because GitHub Actions automatically evaluates the if conditional as an expression. However, this exception does not apply everywhere.

Reply:
that's a cool shorthand